ouroboros-consensus-cardano-legacy-block/ouroboros-consensus-cardano-legacy-block.cabal

```diff
@@ -0,0 +1,74 @@
+cabal-version:   3.0
+name:            ouroboros-consensus-cardano-legacy-block
+version:         0.1.0.0
+synopsis:
+  Legacy blocks for the instantation of the Ouroboros consensus layer used by Cardano
+
+description:
+  Legacy blocks for the instantation of the Ouroboros consensus layer used by Cardano.
+
+license:         Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+
+copyright:       2023-2023 Input Output Global Inc (IOG)
+author:          IOHK Engineering Team
+maintainer:      operations@iohk.io
+category:        Network
+build-type:      Simple
+extra-doc-files:
+  CHANGELOG.md
+  README.md
+
+source-repository head
+  type:     git
+  location: https://github.com/input-output-hk/ouroboros-consensus
+
+flag asserts
+  description: Enable assertions
+  manual:      False
+  default:     False
+
+common common-lib
+  default-language: Haskell2010
+  ghc-options:
+    -Wall -Wcompat -Wincomplete-uni-patterns
+    -Wincomplete-record-updates -Wpartial-fields -Widentities
+    -Wredundant-constraints -Wmissing-export-lists -Wunused-packages
+    -Wno-unticked-promoted-constructors
+
+  if flag(asserts)
+    ghc-options: -fno-ignore-asserts
+
+library
+  import:          common-lib
+  hs-source-dirs:  src
+  exposed-modules:
+    DBSync
+    Legacy.Byron.Ledger
+    Legacy.Cardano.Block
+    Legacy.Cardano.CanHardFork
+    Legacy.LegacyBlock
+    Legacy.Shelley.Ledger
+
+  build-depends:
+    , base                          >=4.14  && <4.17
+    , bytestring
+    , cardano-crypto-class
+    , cardano-ledger-allegra
+    , cardano-ledger-alonzo
+    , cardano-ledger-babbage
+    , cardano-ledger-conway
+    , cardano-ledger-core
+    , cardano-ledger-shelley
+    , cardano-prelude
+    , cardano-protocol-tpraos       >=1.0.1 && <1.1
+    , cardano-slotting
+    , containers                    >=0.5   && <0.7
+    , mtl                           >=2.2   && <2.3
+    , nothunks
+    , ouroboros-consensus           ^>=0.7
+    , ouroboros-consensus-cardano
+    , ouroboros-consensus-protocol  ^>=0.5
```

Comment:
If anything in the consensus/protocol (class) interface changes, it would require changes in this package. And if we add eras to the cardano block, we would have to add an era to the legacy cardano block. So I'm not sure if they are very independent, but I've removed the bounds for now